test/state/account.hpp

```diff
@@ -31,25 +32,32 @@ struct Account
     /// The maximum allowed nonce value.
     static constexpr auto NonceMax = std::numeric_limits<uint64_t>::max();
 
+    static constexpr auto EMPTY_CODE_HASH =
+        0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470_bytes32;
+
     /// The account nonce.
     uint64_t nonce = 0;
 
     /// The account balance.
     intx::uint256 balance;
 
+    bytes32 code_hash = EMPTY_CODE_HASH;
+
+    bool has_cold_storage = false;
+
     /// The account storage map.
-    std::unordered_map<bytes32, StorageValue> storage;
+    std::unordered_map<bytes32, StorageValue> _storage;
```

Comment:
Do you plan to make `_storage` and `code` private / split out of this struct?

Reply:
This was a way to find all usages.